modules/webhook/deliver.go

```diff
@@ -74,13 +74,13 @@ func Deliver(t *models.HookTask) error {
 	}
 
 	req.Header.Add("X-Gitea-Delivery", t.UUID)
-	req.Header.Add("X-Gitea-Event", string(t.EventType))
+	req.Header.Add("X-Gitea-Event", t.EventType.Event())
 	req.Header.Add("X-Gitea-Signature", t.Signature)
 	req.Header.Add("X-Gogs-Delivery", t.UUID)
-	req.Header.Add("X-Gogs-Event", string(t.EventType))
+	req.Header.Add("X-Gogs-Event", t.EventType.Event())
 	req.Header.Add("X-Gogs-Signature", t.Signature)
 	req.Header["X-GitHub-Delivery"] = []string{t.UUID}
-	req.Header["X-GitHub-Event"] = []string{string(t.EventType)}
+	req.Header["X-GitHub-Event"] = []string{t.EventType.Event()}
```

Comment:
Should event types be downgraded for this header (and Gogs')? 🤔

Reply:
I am just making them match what they used to send.